calculation/sum_class.py

```diff
@@ -3,4 +3,8 @@
 
 class Summation(Calculation):
     def get_calculation(self):
```

Comment:
I see that the class is called `Summation` but the file is `sum_class.py`, let's rename this to `summation_class.py` so that we are using the same name for both.